docs/how-to/enable-intel-gpu.rst

```diff
@@ -0,0 +1,124 @@
+.. _enable_intel_gpu:
+
+Enable Intel GPUs
+=============================
+
+This guide describes how to configure DSS to utilise the Intel GPUs on your machine. This can be done by enabling the Intel device plugin on your MicroK8s cluster.
+
+Prerequisites
+-------------
+
+* :ref:`DSS CLI is installed <install_DSS_CLI>` and :ref:`initialised <initialise_DSS>`.
+* Your machine has an Intel GPU.
```

Comment:
```suggestion
* An Intel GPU installed in your machine.
```

Reply:
Are we sure "installed" is a good word here? We want the PC to have it, "install" for me points to software installation.

Reply:
Good point. Let's go with "Your machine includes an Intel GPU". I wasn't happy with using "have" in this context.

Reply:
Agreed